docs/reference/docs/index_.asciidoc

```diff
@@ -99,7 +99,9 @@ The name of the index the document was added to.
 The document type. {es} indices now support a single document type, `_doc`.
 
 `_id`::
-The unique identifier for the added document.
+The unique identifier for the added document. By default, maximum length
```

Comment:
I think mentioning the max limit on the index-modules page is enough, it feels a bit low-level to include here, since a lot of users go here early in their Elasticsearch endeavors.